src/Component/Basket/BasketElement.php

```diff
@@ -202,11 +202,11 @@ public function getUnitPrice($vat = false)
         $price = (string) $this->unitPrice;
 
         if (!$vat && $this->isPriceIncludingVat()) {
-            $price = bcdiv($price, bcadd('1', bcdiv((string) $this->getVatRate(), '100')));
+            $price = bcdiv($price, bcadd('1', (string) bcdiv((string) $this->getVatRate(), '100')));
```

Comment:
some of this is unnecesary, please have a look at the whole type hint thing, the original code already has some (string) because this change was already on master

Reply:
bcdiv can return null `http://php.net/manual/en/function.bcdiv.php#refsect1-function.bcdiv-returnvalues` so this is needed because bcadd expects string?

Reply:
ah, but our divisior is not 0, my bad, will fix it

Reply:
fixed.